GameData/RealismOverhaul/RO_SuggestedMods/Buran-Energia/ENERGIA.cfg

```diff
@@ -0,0 +1,1722 @@
+
+        @PROPELLANT[SolidFuel]
+        {
+            @name = HTPB
+        }
+
+        @atmosphereCurve
+        {
+            @key,0 = 0 205
+            @key,1 = 1 177
+        }
+    }
+	MODULE
+	{
+		name = ModuleFuelTanks
+		volume = 30.6
+		type = HTPB
+		basemass = -1
+	}
+
+    !RESOURCE,*{}
+}
+
+
+@PART[Energia_RDTT3_right_Variant]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+
+    @MODEL
+	{
+		@scale = 1.0, 1.0, 1.0
+	}
+
+	%scale = 1.0
+	%rescaleFactor = 1.5625
+
+    @mass = 0.112
+    @crashTolerance = 10
+	@maxTemp = 873.15
+    %skinMaxTemp = 973.15
+
+    %engineType = RDTT
+    %engineTypeMult = 1
+    %massOffset = 0
+    %ignoreMass = False
+
+    @MODULE[ModuleEngines*]
+    {
+        @name = ModuleEnginesRF
+        @minThrust = 98.0
+        @maxThrust = 98.0
+        @heatProduction = 100
+        @useEngineResponseTime = False
+        @engineAccelerationSpeed = 0
+
+        @PROPELLANT[SolidFuel]
+        {
+            @name = HTPB
+        }
+
+        @atmosphereCurve
+        {
+            @key,0 = 0 205
+            @key,1 = 1 177
+        }
+    }
+	MODULE
+	{
+		name = ModuleFuelTanks
+		volume = 30.6
+		type = HTPB
+		basemass = -1
+	}
+
+    !RESOURCE,*{}
+}
+
+@PART[Energia_RDTT4]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+
+    @MODEL
+	{
+		@scale = 1.0, 1.0, 1.0
+	}
+
+	%scale = 1.0
+	%rescaleFactor = 1.5625
+
+    @mass = 0.112
+    @crashTolerance = 10
+	@maxTemp = 873.15
+    %skinMaxTemp = 973.15
+
+    %engineType = RDTT
+    %engineTypeMult = 1
+    %massOffset = 0
+    %ignoreMass = False
+
+    @MODULE[ModuleEngines*]
+    {
+        @name = ModuleEnginesRF
+        @minThrust = 98.0
+        @maxThrust = 98.0
+        @heatProduction = 100
+        @useEngineResponseTime = False
+        @engineAccelerationSpeed = 0
+
+        @PROPELLANT[SolidFuel]
+        {
+            @name = HTPB
+        }
+
+        @atmosphereCurve
+        {
+            @key,0 = 0 205
+            @key,1 = 1 177
+        }
+    }
+	MODULE
+	{
+		name = ModuleFuelTanks
+		volume = 14.3
+		type = HTPB
+		basemass = -1
+	}
+
+    !RESOURCE,*{}
+}
+
+
+//----------------POLYUS--------------
+
+@PART[POLYUS_SKIF_17F19DM]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+
+    @MODEL
+	{
+		@scale = 1.0, 1.0, 1.0
+	}
+
+	%scale = 1.0
+	%rescaleFactor = 1.5625
+
+    @mass = 61.612
+    @crashTolerance = 10
+	@maxTemp = 873.15
+    %skinMaxTemp = 973.15
+	
+	!RESOURCE[SolidFuel]
+	{
+	}
+    %engineType = RDTT
+    %engineTypeMult = 1
+    %massOffset = 0
+    %ignoreMass = False
+
+    @MODULE[ModuleEngines*]
+    {
+        @name = ModuleEnginesRF
+        @minThrust = 12.0
+        @maxThrust = 12.0
+        @heatProduction = 100
+        @useEngineResponseTime = False
+        @engineAccelerationSpeed = 0
+
+        @PROPELLANT[SolidFuel]
+        {
+//            @name = HTPB
+        }
+
+        @atmosphereCurve
+        {
+            @key,0 = 0 205
+            @key,1 = 1 177
+        }
+    }

```

Comment:
For example, this is the 6th usage of RDTT as `engineType` and where you still declare its stats.